llgtrt/py/phi3.5-vision/input_processor.py

```diff
@@ -0,0 +1,176 @@
+import copy
+import requests
+import llgtrt_base
+import torch
+
+import numpy as np
+import tensorrt_llm
+import subprocess
+
+subprocess.call(["pip", "install", "flash-attn", "--no-build-isolation"])
```

Comment:
please add to docker container setup if need and add "import flash_attn" so that importing plugin fails right away

or else do we actually need this? is it used by the vision part?